src/scanner.c

```diff
@@ -449,30 +456,31 @@ static bool scan(TSLexer* lexer, Stack* stack, const bool* valid_symbols) {
   }
 
   return false;
-
 }
 
 // ---------------------------------------------------------------------------------------
 
-void *tree_sitter_r_external_scanner_create() {
+void* tree_sitter_r_external_scanner_create() {
   bool debug = is_debug_build();
   return stack_new(debug);
 }
 
-bool tree_sitter_r_external_scanner_scan(void *payload,
-                                         TSLexer *lexer,
-                                         const bool *valid_symbols) {
+bool tree_sitter_r_external_scanner_scan(
+    void* payload, TSLexer* lexer, const bool* valid_symbols
+) {
```

Comment:
Can we configure clang-format to prefer vertical arguments for this sort of code?

Reply:
<img width="591" alt="Screenshot 2024-03-07 at 12 23 47 PM" src="https://github.com/r-lib/tree-sitter-r/assets/19150088/be232f40-aa1a-4457-aceb-4947546a3bfd">

I can make it looks like this with a combination of:

```
BinPackParameters: false
AllowAllParametersOfDeclarationOnNextLine: false
```

How's that?

Reply:
Thanks; this looks good to me!